packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -57,12 +57,20 @@ function printAttributes(attributes, print, indent, colors, opts) {
 }
 
 const print = (
-  element: any,
+  element: HTMLElement | Text | Comment,
   print: Print,
   indent: Indent,
   opts: Options,
   colors: Colors,
-) => {
+): string => {
+  if (element instanceof Text) {
```

Comment:
Not sure if you want these`instanceof` checks, but it was the only way I could get `flow` to accept my input. Will it work outside of JSDOM? Do we care?